gnucash/gnome-utils/gnc-tree-view-account.c

```diff
@@ -935,12 +949,13 @@ gnc_tree_view_account_new_with_root (Account *root, gboolean show_root)
                                            GNC_TREE_VIEW_COLUMN_VISIBLE_ALWAYS,
                                            sort_by_total_value);
 
-    gnc_tree_view_add_numeric_column(view, _("Total (Period)"), "total-period",
-                                     SAMPLE_ACCOUNT_VALUE,
-                                     GNC_TREE_MODEL_ACCOUNT_COL_TOTAL_PERIOD,
-                                     GNC_TREE_MODEL_ACCOUNT_COL_COLOR_TOTAL_PERIOD,
-                                     GNC_TREE_VIEW_COLUMN_VISIBLE_ALWAYS,
-                                     sort_by_total_period_value);
+    column = gnc_tree_view_add_numeric_column(view, _("Total (Period)"), "total-period",
+                                              SAMPLE_ACCOUNT_VALUE,
+                                              GNC_TREE_MODEL_ACCOUNT_COL_TOTAL_PERIOD,
+                                              GNC_TREE_MODEL_ACCOUNT_COL_COLOR_TOTAL_PERIOD,
+                                              GNC_TREE_VIEW_COLUMN_VISIBLE_ALWAYS,
+                                              sort_by_total_period_value);
+    gnc_tree_view_set_column_cell_renderer_pango_wrap_word (column);
```

Comment:
What does word-wrapping numeric columns do?

Reply:
It sets the cell render to wrap on word and also sets the wrap-width to that of the column width. Setting the wrap-width enables the wrapping, setting it to -1 does not fix the problem.

Reply:
Yes of course. But what does it do for ***numeric*** columns that don't have words and shouldn't wrap?

Reply:
I under stand what you mean now, does not look good, will have a rethink over the weekend, maybe look at changing the original rtl fix.